test/e2e/actions/project_flow.go

```diff
@@ -0,0 +1,27 @@
+package actions
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions/deploy"
+	kubecli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/kubecli"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/config"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/model"
+)
+
+func ProjectCreationFlow(userData *model.TestDataProvider) {
+	By("Deploy Project with requested configuration", func() {
+		PrepareUsersConfigurations(userData)
+		deploy.NamespacedOperator(userData) // TODO: how to deploy operator by code?
```

Comment:
Something left to do?

Reply:
Not at the moment. This is the part that still uses kubectl to deploy operator and crd.